src/lib/merkle_ledger_tests/test_mask.ml

```diff
@@ -262,9 +264,14 @@ module Make (Test : Test_intf) = struct
     if Test.depth <= 8 then
       Test.with_instances (fun maskable mask ->
           let attached_mask = Maskable.register_mask maskable mask in
-          Mask.Attached.set attached_mask dummy_location dummy_account ;
+          let loc0 =
+            Test.Location.Addr.of_directions
+              (List.init Test.depth ~f:(fun _ -> Direction.Left))
```

Comment:
Any reason for using this over the more interesting `Left;Right;Left...`?

Reply:
Yes, the leftmost one has index 0, which allows putting as many accounts as possible in the ledger. It's also easier to calculate that number. We use the alternating path in other tests.